API/Osu/AutomationChecks/GameAutomationChecks.cs

```diff
@@ -79,122 +80,70 @@ public static bool PassesTeamSizeCheck(Game game)
             return false;
         }
 
-        var teamVs = game.TeamType == TeamType.TeamVs;
-        if (teamSize == 1 && !teamVs)
+        if (teamSize == 1)
         {
-            var countPlayers = game.MatchScores.Count;
-            var refereePresent = game.MatchScores.Any(score => score.Score <= AutomationChecksUtils.MinimumScore);
-
-            var countAfterReferees = countPlayers - game.MatchScores.Count(score => score.Score <= AutomationChecksUtils.MinimumScore);
-
-            var satisfiesOneVersusOne = refereePresent ? countAfterReferees == 2 : countPlayers == 2;
+            var satisfiesOneVersusOne = validScores.Count == 2;
             if (!satisfiesOneVersusOne)
             {
                 s_logger.Information(
-                    "{Prefix} Match {MatchId} has a team size of 1, but does not satisfy the 1v1 checks, can't verify game {GameId} [had {CountPlayers} players | Referee: {Ref}]",
+                    "{Prefix} Match {MatchId} has a team size of 1, but does not satisfy the 1v1 checks, can't verify game {GameId}",
                     LogPrefix,
                     game.Match.MatchId,
-                    game.GameId,
-                    countPlayers,
-                    refereePresent
+                    game.GameId
                 );
             }
 
             return satisfiesOneVersusOne;
         }
 
-        // teamSize is greater than 1 or is TeamVS
+        var countRed = validScores.Count(s => s is { Team: (int)Team.Red });
```

Comment:
```suggestion
        var countRed = validScores.Count(s => s.Team == Team.Red);
```
Don't see the need to use an object comparison here

Reply:
Had it before to cover multiple props. Fixed.